### PR TITLE
Adding sponsorship prospectus

### PR DESCRIPTION
**Pull request by arschles**

### CODE REVIEW

File: PROSPECTUS.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+- 8 tickets (max 2 at table at any time)
+- 5 minute “promo’ spot during talk intermissions
+- Table presence at the venue
+- Logo on site and conference materials
+
+## Silver: $1,000 (Limit 10)
+
+- 6 tickets (max 2 at a table)
+- 10-minute “promo’ spot during talk intermissions
+- Table presence at the venue
```
Comment (jberkus) [inline PROSPECTUS.md]: Shared table presence (silver sponsors get 1/2 table)

File: PROSPECTUS.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Kubernetes Community Days Portland Sponsorship Opportunies
+
+_Adapted from [the Bouvet Island prospectus](https://kubernetescommunitydays.org/events/2020-bouvet-island/sponsor/)_
+
+## Post-Event Social: Contact us (Limit 1)
+
+- Opportunity to sponsor a fully branded post-event social
+- 8 Tickets
```
Comment (jberkus) [inline PROSPECTUS.md]: I want to lower the numbers of tickets; I don't want to hold most of the tickets off the market until 2 weeks before the event.  My suggestion:

Gold/Event: 5 tix, Silver: 2 tix

File: PROSPECTUS.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Kubernetes Community Days Portland Sponsorship Opportunies
+
+_Adapted from [the Bouvet Island prospectus](https://kubernetescommunitydays.org/events/2020-bouvet-island/sponsor/)_
+
+## Post-Event Social: Contact us (Limit 1)
```
Comment (jberkus) [inline PROSPECTUS.md]: We know the amount, it's $4000. 

I think we should also allow for 2 sponsors to split this sponsorship.

Comment (jberkus) [reply PROSPECTUS.md]: Per Eric, increase to $5000

File: PROSPECTUS.md (hunk 2 above)
Comment (jberkus) [reply PROSPECTUS.md]: Per meeting, 6 tix for event sponsorship

File: PROSPECTUS.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+- Opportunity to sponsor a fully branded post-event social
+- 8 Tickets
+- Table presence at the venue
+- Logo on site and conference materials
+
+## Gold: #3,000 (Limit 3)
+
+- Logo on event materials and website
+- 8 tickets (max 2 at table at any time)
```
Comment (jberkus) [inline PROSPECTUS.md]: see above

File: PROSPECTUS.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+- 5 minute “promo’ spot during talk intermissions
```
Comment (jberkus) [inline PROSPECTUS.md]: 2 minutes

File: PROSPECTUS.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+- Table presence at the venue
+- Logo on site and conference materials
+
+## Silver: $1,000 (Limit 10)
+
+- 6 tickets (max 2 at a table)
+- 10-minute “promo’ spot during talk intermissions
```
Comment (jberkus) [inline PROSPECTUS.md]: suggest that silver don't get a shoutout

File: PROSPECTUS.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+- Logo on event materials and website
+- 8 tickets (max 2 at table at any time)
+- 5 minute “promo’ spot during talk intermissions
+- Table presence at the venue
+- Logo on site and conference materials
+
+## Silver: $1,000 (Limit 10)
+
+- 6 tickets (max 2 at a table)
```
Comment (jberkus) [inline PROSPECTUS.md]: see above

File: PROSPECTUS.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+- Table presence at the venue
+- Logo on site and conference materials
+
+## Gold: #3,000 (Limit 3)
+
+- Logo on event materials and website
+- 8 tickets (max 2 at table at any time)
+- 5 minute “promo’ spot during talk intermissions
+- Table presence at the venue
+- Logo on site and conference materials
```
Comment (jberkus) [inline PROSPECTUS.md]: Add:

- Thank you in opening and closing presentations
- Mention in all conference-related announcements
- Logo on banner at the conference site

File: PROSPECTUS.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+_Adapted from [the Bouvet Island prospectus](https://kubernetescommunitydays.org/events/2020-bouvet-island/sponsor/)_
+
+## Post-Event Social: Contact us (Limit 1)
+
+- Opportunity to sponsor a fully branded post-event social
+- 8 Tickets
+- Table presence at the venue
+- Logo on site and conference materials
+
+## Gold: #3,000 (Limit 3)
```
Comment (tpepper) [inline PROSPECTUS.md]: typo: "$3,000"

File: PROSPECTUS.md [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+- 5 minute “promo’ spot during talk intermissions
+- Table presence at the venue
+- Logo on site and conference materials
+
+## Silver: $1,000 (Limit 10)
+
+- 6 tickets (max 2 at a table)
+- 10-minute “promo’ spot during talk intermissions
+- Table presence at the venue
+- Logo on conference materials
```
Comment (jberkus) [inline PROSPECTUS.md]: Add:

- Thank you in opening and closing presentations

Comment (jberkus) [reply PROSPECTUS.md]: SHould we offer name on banner?  Maybe smaller?

File: PROSPECTUS.md [hunk 11]
```diff
@@ -0,0 +1,25 @@
+# Kubernetes Community Days Portland Sponsorship Opportunies
+
+_Adapted from [the Bouvet Island prospectus](https://kubernetescommunitydays.org/events/2020-bouvet-island/sponsor/)_
+
+## Post-Event Social: Contact us (Limit 1)
+
+- Opportunity to sponsor a fully branded post-event social
```
Comment (jberkus) [inline PROSPECTUS.md]: Exclusive branding on social event

**Comment by tpepper** [comment]
lgtm

**Review by jessedearing — APPROVED** [review]

File: PROSPECTUS.md [hunk 12]
```diff
@@ -0,0 +1,30 @@
...
+_Adapted from [the Bouvet Island prospectus](https://kubernetescommunitydays.org/events/2020-bouvet-island/sponsor/)_
+
+## Post-Event Social: $6,000
+
+_Two sponsors may split this sponsorship. In case of a split, each sponsor get all benefits below in full unless otherwise noted._
+
+- Exclusive branding at event
+- 6 Tickets
+  - 3 tickets per sponsor if two sponsors split
+- Table presence at the venue
```
Comment (jberkus) [inline PROSPECTUS.md:12]:
```suggestion
- Table presence at the venue (shared if two)
```

**Pull request merged**